src/components/APIStarter.jsx

```diff
@@ -35,9 +390,21 @@ const APIStarter = () => {
     //     );
     //   })}
     // </ul>
-    <>{ response.length==0 ? <div></div>: (<div>
```

Comment:
Use `===` while checking a condition.

Why are you rendering empty `div` unnecessary when `response.length === 0`, returning `null` makes more sense here.